management/src/main/java/nearsoft/skytouch/management/TaskApplication.java

```diff
@@ -1,12 +1,19 @@
 package nearsoft.skytouch.management;
 
+import nearsoft.skytouch.common.ProductJSONSerializer;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
 
 @SpringBootApplication
 public class TaskApplication {
 
     public static void main(String[] args) {
         SpringApplication.run(TaskApplication.class, args);
     }
+
+    @Bean
+    public ProductJSONSerializer getProductJSONSerializer() {
+        return new ProductJSONSerializer();
```

Comment:
You can annotate this class as a @Component, and let Spring create the bean and wire everything up. You only need to create beans manually when you need to create it with an specific state.

Reply:
Ok thanks!